### DISCUSSION TITLE
XDM Browser Monitor is too much of the same option!

### DISCUSSION THREAD

**Discussion by erenkaya123456789** [subhra74/xdm#653] [Q&A] [answered]
XDM Browser Monitor comes up with too many of the same options, how can I know which one is right or how can I eliminate it to the right option?
![Ekran görüntüsü 2021-09-22 062710](https://user-images.githubusercontent.com/82344100/134380711-410a1784-0fb7-487c-bdf7-a7491cf152bf.png)
![Ekran görüntüsü 2021-09-22 062800](https://user-images.githubusercontent.com/82344100/134380743-0ae2a735-c0fc-4e76-9c9b-9ff9ae1fd013.png)

**Comment by subhra74** [comment, marked as answer]
Although the file name and resolution is same, the bitrate is different. High bitrate=better quality but bigger size
